support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/new-flow/ContributionThankYouArticleShare.jsx

```diff
@@ -0,0 +1,47 @@
+// @flow
+// $FlowIgnore - required for hooks
+import React, { useEffect, useState } from 'preact/compat';
+import ActionContainer from './components/ActionContainer';
+import ActionHeader from './components/ActionHeader';
+import ActionBody from './components/ActionBody';
+import SvgShare from './components/SvgShare';
+import ShareableArticleContainer from './components/ShareableArticleContainer';
+
+type PropTypes = { header: string, body: string }
+
+const ContributionThankYouArticleShare = (props: PropTypes) => {
+  const [articleData, setArticleData] = useState(null);
+
+  useEffect(() => {
+    fetch('/thank-you-shareable-articles')
+      .then(response => response.json())
+      .then(data => setArticleData(data));
+  }, []);
+
+  const actionIcon = <SvgShare />;
+  const actionHeader = <ActionHeader title={props.header} />;
+  const actionBody = (
+    <ActionBody>
+      <p>{props.body}</p>
+      {articleData && articleData.map(article => (<ShareableArticleContainer
+        articleUrl={article.articleUrl}
+        headline={article.headline}
+        imageUrl={article.image.imageUrl}
+        imageAltText={article.image.altText}
+      />))}
```

Comment:
Might be worth adding a key to satisfy react

Reply:
I'm not sure I understand @TomPretty, can you post an example of what you had in mind?

Reply:
Yeah for sure! I don't think it will actually matter here, but react will complain when you map over a list and don't pass in a `key` prop. Just to satisfy react you can do:

```jsx
{articleData && articleData.map(article => (<ShareableArticleContainer
        key={article.Url} // just needs to be something unique for that article 
        articleUrl={article.articleUrl}
        headline={article.headline}
        imageUrl={article.image.imageUrl}
        imageAltText={article.image.altText}
      />))}
```

Reply:
Ah, cool. That's news to me, so thank for the tip! I've added the key prop now ✅